docs/index.rst

```diff
@@ -123,25 +129,42 @@ and some of the core features of Read the Docs.
    /faq
 
 
-Step-by-step Guides
--------------------
+How-to Guides
+-------------
 
 These guides will help walk you through specific use cases
 related to Read the Docs itself, documentation tools like Sphinx and MkDocs
 and how to write successful documentation.
 
-* :doc:`/guides/tools`
-* :doc:`/guides/platform`
-* :doc:`/guides/commercial`
+* **For documentation authors**:
+  :doc:`/guides/cross-referencing-with-sphinx` |
+  :doc:`/guides/intersphinx` |
+  :doc:`/guides/jupyter` |
+  :doc:`Other guides for authors </guides/authors>`
```

Comment:
```suggestion
  :doc:`Additional guides for authors </guides/authors>`
```

Maybe better than `Other`?  There also aren't any other guides there, but hopefull we'll fix that :)

Reply:
Good point! I'll replace it with `More`